src/webAgentAPI/main.ts

```diff
@@ -1,15 +1,18 @@
 import { createAPI, setListener } from "./api";
```

Comment:
NIT: I think the `setListener` can be removed as it's no longer used in this file. Also, I think that the export no longer exists in the `api` file.

Reply:
good catch.  fixed